### ISSUE TITLE
Converting .pth files

### ISSUE THREAD

**Issue by dridri** [balisujohn/tortoise.cpp#19]
Hello,
I found a fine-tuned model to handle French : https://huggingface.co/Snowad/French-Tortoise , but the files are in Torch format. Is there any way to convert it to ggml's format ?

**Comment by balisujohn** [comment]
Yeah I'll publish those scripts in the next few days. Let me know if you decide to publish the converted files for the french model and I can link to them in the readme.

**Comment by balisujohn** [comment]
I'm also going to try converting it myself out of curiosity.

**Comment by baileyheading** [comment]
I'd be keen to try the result, as a French speaker

**Comment by tuanh123789** [comment]
hi @balisujohn can you provide script to convert tortoise to ggml format, thank you

**Comment by eloop001** [comment]
> hi [@balisujohn](https://github.com/balisujohn) can you provide script to convert tortoise to ggml format, thank you

`import torch
import numpy as np

voice = "your_voice.pth"  # Replace with your actual file path

pth_file = torch.load(voice, map_location=torch.device('cpu'))

tensor_to_convert = pth_file[0] 
numpy_array = tensor_to_convert.numpy()

file_path = voice.replace('.pth', '.bin')

# Save NumPy array as binary file
numpy_array.tofile(file_path)

exit()`
